src/styles/stars.scss

```
@@ -0,0 +1,32 @@
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  margin-inline: $margin-inline;
+
+  &_stars-block {
+    display: flex;
+    flex-direction: row;
+  }
+
+  &_star {
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &_star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &_text {
```

and move to `card.scss` file
```suggestion
  .card__reviews {
```

---

ok